src/main/java/dev/dirs/Util.java

```diff
@@ -148,13 +153,19 @@ static String linuxExecutableDir(String homeDir, String dataDir) {
     }
 
     String encodedCommand = SCRIPT_START_BASE64 + toUTF16LEBase64(buf.toString() + "}");
-
-    return runCommands(guidsLength, Charset.forName("UTF-8"),
-        "powershell.exe",
-        "-NoProfile",
-        "-EncodedCommand",
-        encodedCommand
-    );
+    String[] commands = new String[] { "pwsh.exe", "powershell.exe" };
+    for (String command : commands) {
+      try {
+        return runCommands(guidsLength, Charset.forName("UTF-8"),
+            command,
+            "-NoProfile",
+            "-EncodedCommand",
+            encodedCommand
+        );
+      } catch (IOException e) {
```

Comment:
Wouldn't it be better to not swallow this? Maybe by adding the latest exception as a cause to the `RuntimeException` below?

Reply:
The reason I swallowed it was because I don't think it's actionable on the part of the user.

Reply:
I also wasn't sure which `IOException` should be reported if both commands failed. I wouldn't mind arbitrarily using the first failure as the cause of the `RuntimeException` if you would prefer that.